stories/core/ColorPicker.stories.tsx

```diff
@@ -0,0 +1,178 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React, { useState } from 'react';
+import {
+  ColorPicker,
+  ColorPickerProps,
+  IconButton,
+  Tooltip,
+} from '../../src/core';
+import { action } from '@storybook/addon-actions';
+import Color from '../../src/core/ColorPicker/Color';
+import { DefaultColors } from '../../src/core/ColorPicker/ColorPicker';
+
+export default {
+  component: ColorPicker,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    children: { control: { disable: true } },
+  },
+  title: 'Core/ColorPicker',
+} as Meta<ColorPickerProps>;
+
+export const Basic: Story<ColorPickerProps> = (args) => {
+  const { activeColorIndex = 0 } = args;
+
+  const [opened, setOpened] = useState(false);
+
+  const [currentColorName, setCurrentColorName] = useState(
+    DefaultColors[activeColorIndex].props['name'],
+  );
+  const [currentColorValue, setCurrentColorValue] = useState(
+    DefaultColors[activeColorIndex].props['color'],
+  );
+
+  const onClick = (index: number) => {
+    action(`Clicked color #${index}`)();
+    setCurrentColorName(DefaultColors[index].props['name']);
```

Comment:
Better save index and reuse it. Also, you need to pass it to the `ColorPicker` because when you close and reopen it always show blue cola.

Reply:
This is fixed now, when you close and open the color picker the selected color remains